vcs/gogpm.go

```diff
@@ -73,7 +74,30 @@ func (p *PackageRepo) RootImportPath() string {
 }
 
 func (p *PackageRepo) Dir() string {
-	return filepath.Join(os.Getenv("GOPATH"), "src", p.rr.root)
+	// split gopath
+	goPath := os.Getenv("GOPATH")
+	paths := strings.Split(goPath, ":")
```

Comment:
Thanks @shurcooL. I wasn't aware of that platform difference (and the helper func), will put in another PR to fix.